CHANGELOG.md

```diff
@@ -80,6 +80,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   we now try best to print everything we can, and print an appropriate error message
   instead of crashing.
 - We no longer produce duplicate SMT assertions regarding concrete keccak values.
+- Ord is now correctly implemented for Prop.
```

Comment:
Yes. Sorry :)

Reply:
This might be optimistic from me :)
Time will tell...